sklearn/decomposition/_nmf.py

```diff
@@ -1245,23 +1251,41 @@ def fit(self, X, y=None, **params):
         self.fit_transform(X, **params)
         return self
 
-    def inverse_transform(self, W):
+    def inverse_transform(self, Xt=None, W=None):
         """Transform data back to its original space.
 
         .. versionadded:: 0.18
 
         Parameters
         ----------
-        W : {ndarray, sparse matrix} of shape (n_samples, n_components)
+        Xt : {ndarray, sparse matrix} of shape (n_samples, n_components)
             Transformed data matrix.
 
+        W : deprecated
+            Use `Xt` instead.
+
+            .. deprecated:: 1.3
+
         Returns
         -------
         X : {ndarray, sparse matrix} of shape (n_samples, n_features)
             Returns a data matrix of the original shape.
         """
+        if Xt is None and W is None:
+            raise TypeError("Missing required positional argument: Xt")
```

Comment:
feels a bit weird but I think it's fine since it's only during the deprecation cycle.